### PR TITLE
Show error on Profile when unable to fetch all required data

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Show a general error on top of all the Profile pages if one of the required endpoints errors out.

## Testing done
Added RTL tests to ensure the error is shown when it should be.

## Screenshots
**Error when full name can't load**
image

**When personal info cannot load**
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx [hunk 1]
```diff
@@ -3,35 +3,19 @@ import PropTypes from 'prop-types';
 import { Prompt } from 'react-router-dom';
 import { connect } from 'react-redux';
 
-import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx:6]: moved the alert from this component up to the Profile2Wrapper component so it appears on all of the Profile pages.

**Comment by erikphansen** [comment]
@tressaellen can you review the attached screenshots?

**Comment by erikphansen** [comment]
Looks like this introduced some Cypress failures. I think I know the problem...

**Comment by tressaellen** [comment]
@erikphansen this looks good to me!

**Review by sanlouise — APPROVED** [review]
Great job! 🔥 👍

**Pull request merged**